### ISSUE TITLE
HTTP 400 unsupported path "/query" error with latest clickhouse-odbc

### ISSUE THREAD

**Issue by morokin** [Vertamedia/chproxy#42]
chproxy is not working with https://github.com/yandex/clickhouse-odbc/releases/tag/v1.0.0.20190201 driver

**Comment by valyala** [comment]
- Did it work with the previous versions of `clickhouse-odbc` driver?
- Which error does `chproxy` log on `clickhouse-odbc` access?
- Which error does `clickhouse-odbc` driver return on error?

**Comment by morokin** [comment]
It works with the previous release without issues. 

Which error does clickhouse-odbc driver return on error?
I apologize, seems I wasn't clear enough, I posted exact error in head of issue, so here it is error returned from odbc driver:

HTTP 400 unsupported path "/query" 

I will post debug output from chproxy soon

**Comment by morokin** [comment]
```
Mar 08 21:06:15 hostname chproxy[2766]: ERROR: 2019/03/08 21:06:15 main.go:246: "ip:51577": unsupported path: "/query"
```

This is only output I see in chproxy logs while I'm attempting to run any query from application using latest ODBC driver. Everything works fine with previous driver as I mentioned above

**Comment by valyala** [comment]
The issue should be fixed starting from [v1.13.1](https://github.com/Vertamedia/chproxy/releases).

**Comment by morokin** [comment]
It works, appreciate your quick response.
